File: Pramp-Solutions/decrypt-message.cpp
#include <iostream>
#include <string>

using namespace std;
string decrypt(const string& word){
    string res = "";
    int val = 1;
    for (int i = 0; i < word.size(); i++) {
        char ec = word[i];
        char dc = ec - val;
        while (dc < 'a') dc += 26;
        res.push_back(dc);
        val = ec;
    }
    return res;
}

int main() {
    return 0;
}

File: Pramp-Solutions/decrypt-message.java
import java.io.*;
import java.util.*;

class Solution {
    static String decrypt(String word) {
        String s = "";
        int oldValue = 1, newValue;
        for (int i = 0; i < word.length(); i++) {
            newValue = word.charAt(i) - oldValue;
            oldValue = word.charAt(i);
            while (newValue < 'a') {
                newValue += 26;
            }
            s = s + (char) newValue;
        }
        return s;
    }

    public static void main(String[] args) {

    }

}